### ISSUE TITLE
Webstorm 2020.3 - not working

### ISSUE THREAD

**Issue by PLSFIX** [styled-components/webstorm-styled-components#97]
* **IDE name and version:** 2020.3

* **Styled-components plugin version:** 203.7148.54

## Problem description:
Plugin doesn't work on latest Webstorm at all. 
<img width="142" alt="Screenshot at Mar 11 21-31-15" src="https://user-images.githubusercontent.com/8660125/110836539-3a11cf00-82b1-11eb-93af-d2390c9aa550.png">

**Comment by prigara** [comment]
Hello @PLSFIX, sorry, I can't reproduce the problem in the .tsx file in WebStorm 2020.3.2.
Does the problem occur in all projects or only in this specific project? Could you please share the IDE logs (menu Help - Collect logs) and a sample project?

**Comment by PLSFIX** [comment]
Hello, @prigara. I've tested it on another clean project with the same result.
[test-styled.zip](https://github.com/styled-components/webstorm-styled-components/files/6132387/test-styled.zip) - example project (basically, linaria & 2 files)
[webstorm-logs-20210312-210653.zip](https://github.com/styled-components/webstorm-styled-components/files/6132399/webstorm-logs-20210312-210653.zip) - logs

comparing with VSCode:
<img width="1682" alt="Screenshot at Mar 12 21-06-19" src="https://user-images.githubusercontent.com/8660125/110987163-be328800-837f-11eb-980f-a6ff84755b08.png">

**Comment by PLSFIX** [comment]
Update: it works on Windows
![Screenshot_2](https://user-images.githubusercontent.com/8660125/111067011-0d3d0200-84d3-11eb-8a8b-ad46358d134e.jpg)

**Comment by vladsholokhov** [comment]
Hello! @PLSFIX 
I found the following line in your logs:
```2021-03-11 21:14:08,273 [   1695]   WARN - llij.ide.plugins.PluginManager - Problems found loading plugins:
  The Styled Components & Styled JSX (id=com.deadlock.scsyntax, 
path=/Applications/WebStorm.app/Contents/plugins/styled-components) 
plugin Requires plugin 'org.jetbrains.plugins.less' to be enabled.

Does the problem still exist if you enable Less plugin?

**Comment by PLSFIX** [comment]
Hello @vladsholokhov 
Yes, currently LESS plugin is active but the situation is still the same.

**Comment by PLSFIX** [comment]
Problem has been fixed with the last IDE update `WebStorm 2020.3.3 Build #WS-203.7717.59, built on March 15, 2021`

**Comment by vladsholokhov** [comment]
@PLSFIX 
Unfortunately I had no luck reproducing the issue. 
Thanks for the update!
